footprints/templates/pathmapper/map.html

```diff
@@ -93,6 +98,8 @@ <h2>Share results</h2>
     <script type="text/javascript"
         src="//maps.google.com/maps/api/js?key={{settings.GOOGLE_MAP_API}}&libraries=places"></script>
 
+    <script src="https://code.highcharts.com/highcharts.src.js"></script>
```

Comment:
Might be good to pin this URL to a more specific version - it looks like this works: https://code.highcharts.com/8.1/highcharts.src.js

Reply:
Great point. I've updated.